Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/Implementation/Experimental/ExperimentalFeaturesTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Experimental
+{
+    [TestClass]
+    public class ExperimentalFeaturesTests
+    {
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtensionIfConfigEmpty()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration();
+
+            bool value = telemetryConfiguration.EvaluateExperimentalFeature("abc");
+            Assert.IsFalse(value);
+        }
+
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtension()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration
+            {
+                ExperimentalFeatures = { "abc" }
+            };
+
+            bool value = telemetryConfiguration.EvaluateExperimentalFeature("abc");
+            Assert.IsTrue(value);
+
+            bool fakeValue = telemetryConfiguration.EvaluateExperimentalFeature("fake");
+            Assert.IsFalse(fakeValue);
+        }
+
+        [TestMethod]
+        public void VerifyCanAddExperimentalFeatures()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration
+            {
+                ExperimentalFeatures = { "abc" }
```

Comment:
can you remove this line to really test that no null ref is thrown when you do .Add("123")